index.html

```diff
@@ -1,22 +1,19 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins:wght@200;400;600&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
-    />
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins:wght@200;400;600&display=swap"
+      rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
     <link rel="stylesheet" href="style.css" />
   </head>
+
   <body>
     <header class="navbar">
       <nav>
```

Comment:
- [ ] You still have the same issue 😅 I think it will be solved by simply adding equal padding to left and right in the navbar 👍 this way, you will have the same space around the navbar horizontally 💯

## Your design:
![image](https://user-images.githubusercontent.com/50721479/199711605-77e9a810-d855-45e9-99ca-59b2d83e1c39.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/199711637-7210587f-1b35-4065-bfce-bc5bf47688ad.png)

Reply:
Kindly check again because probably you haven't paid attention to the changes . 
https://www.loom.com/share/52bccaee5b99473bbb6bf1eb65f92dc4   here is a video supporting what I said.